### ISSUE TITLE
Better handling of no-article-found case

### ISSUE THREAD

**Issue by maxkfranz** [PathwayCommons/factoid#1312]
## Description

**Q: What is the name of the feature?**

A: Better handling of no-article-found case

**Q: What does this feature enable the user to do?**

A: Enter article details when article not found in Pubmed / Biorxiv.

**Q: What information must the user provide to use the feature?**

A: 

- Author list / first & last (free text)
- Article title (already provided)
- Journal name
- Year

**Q: What are the applicable constraints, e.g. compatibility or performance?**

A: 

- Everything but the author list is simple
- The author list could be formatted in lots of different ways (e.g. comma-separated, semicolons, initials, middle name...)
- Just do free text (good for display only) for author list and show as-is until populated by cron 

**Q: How does this feature affect each class of user (persona)?**

A:

- Biologist: 
- Editor: 
- Computational biologist: 
- Curator:

**Comment by jvwong** [comment]
As we are piloting with Nature Cell Biology, I looked back at their last 3 issues (Volume 26, Issues 9-11) to get a sense of the relationship between preprints and the version of record/publication. In this period, where were N=34 research articles published in this period. 

**The take home message** Most published NCB research articles won't have a preprint, and even if it does, the titles won't be the same. Only 4/34 articles had a preprint with the same title. You mileage may vary depending on the journal.

Notes:
  - PubMed and Crossref lack any information about relations between the preprint and publication, at least at the point of publication
  - CrossRef provides information and code used to [match preprints-articles](https://www.crossref.org/blog/discovering-relationships-between-preprints-and-journal-articles/#:~:text=The%20Crossref%20deposit%20schema%20allows,relationship%20deposited%20with%20a%20preprint)
    - Incredibly accurate tool to match DOI to preprint (100%): https://marple.research.crossref.org/match?task=preprint-matching&strategy=preprint-sbmv&input=10.1038/s41556-024-01541-5
    - Can we extract this, get the converse service: preprint --> publication?

--- 

**1. How long does it take for an accepted NCB article to end up on PubMed?**

<img width="450px" src="https://github.com/user-attachments/assets/3d4e34f0-efa5-4db4-bece-d4dd72a2dbf6" />

**2. What proportion of NCB articles had an associated preprint?**

<img height="300px" src="https://github.com/user-attachments/assets/e68a46a1-a25a-4097-b2eb-571b074a38d9" />

**3. For NCB articles with a preprint (N=15), what is the period between the posting and PubMed indexing?**

<img width="450px" src="https://github.com/user-attachments/assets/f2c70306-9c57-4181-b1b2-acae55f354bf "/>

**4. For NCB articles with a preprint, how often are the titles identical?**

<img width="350px" src="https://github.com/user-attachments/assets/dec354f0-5085-4b49-9e4e-0bf1fcfe3d84" />

**Comment by maxkfranz** [comment]
@jvwong

Hi, Jeff.  I'll take a look at this one next week.  Keep an eye out for the PR.

**Comment by jvwong** [comment]
> @jvwong
> 
> Hi, Jeff. I'll take a look at this one next week. Keep an eye out for the PR.

OK - before the break I looked at the idea of a 'self-contained' journal search box. 

https://github.com/PathwayCommons/factoid/compare/unstable...iss1312_journal-input

I think the most useful part of this is (not shown) rather detailed list of journals synonyms and ID (aka ISSN code). Maybe some of this is useful.

**Comment by maxkfranz** [comment]
Yeah, that makes sense.  I think validation/autocomplete isn't strictly necessary though, since eventually the cron job will clean it up anyway.  It would be nice to have to make things quicker, though (less typing).

**Comment by maxkfranz** [comment]
I'll start with the minimal simple version and then we can take it from there.

**Comment by jvwong** [comment]
This is the node CML script that produces a list of journals, their synonyms and ISSNs. I manually added some preprints. I downloaded the raw data from https://www.scimagojr.com/. PubMed produces some file downloads for this sorta thing but frankly they were useless in practice.

https://github.com/jvwong/journal-list/blob/main/data/journal-list.json
